### PR TITLE
Only apply triggers for the master branch

### PR DESCRIPTION
**Pull request by stephendonner**
@chartjes @davehunt r?

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -12,8 +12,8 @@ pipeline {
     KINTO_DOTENV = credentials('KINTO_DOTENV')
   }
   triggers {
-    pollSCM('H/5 * * * *')
-    cron('H H * * *')
+    pollSCM(env.BRANCH_NAME == 'master' ? 'H/5 * * * *' : '')
+    cron(env.BRANCH_NAME == 'master' ? 'H * * * *' : '')
```
Comment (davehunt) [inline Jenkinsfile]: Did you mean to change this from daily to hourly?

File: Jenkinsfile [hunk 2]
```diff
@@ -12,8 +12,8 @@ pipeline {
     KINTO_DOTENV = credentials('KINTO_DOTENV')
   }
   triggers {
-    pollSCM('H/5 * * * *')
-    cron('H H * * *')
+    pollSCM(env.BRANCH_NAME == 'master' ? 'H/5 * * * *' : '')
```
Comment (davehunt) [inline Jenkinsfile]: I'd vote for removing this, as multibranch pipeline will detect changes when it scans.

Comment (davehunt) [reply Jenkinsfile]: On second thoughts, mutlibranch is only in use on preprod, so maybe let's keep it.

File: Jenkinsfile (hunk 1 above)
Comment (stephendonner) [reply Jenkinsfile]: No, thanks for spotting that, and sorry for missing it!

**Pull request merged**